### ISSUE TITLE
Trying to save edits on a kata results in "You do not have permissions to view this page" error message

### ISSUE THREAD

**Issue by FArekkusu** [codewars/codewars.com#2525]
While trying to enable Python 3.8 on [this kata](https://www.codewars.com/kata/546c7f89bed2e12fb300056f) I've run into a problem: after making some changes to the final tests and hitting `Re-Publish` the "Please wait while we publish" pop-up appeared but then very quickly the contents of the kata editor page were replaced with an error message saying "You do not have permissions to view this page".

![image](https://user-images.githubusercontent.com/31668662/140902786-0c860536-b7ff-431a-b105-9a1c34ca9b96.png)

I remember I've encountered this message when trying to view revisions of a kata that I was not a contributor of (by manually entering the kata's revisions page URL in browser), but I don't see what could cause this message to appear on a kata which has had issues open for ~7 years.

**Comment by Blind4Basics** [comment]
do you still see a rank assigned, in the editor?

**Comment by FArekkusu** [comment]
No. IIRC this breaks republishing, but has it always resulted in the "You do not have permissions..." message?

**Comment by Blind4Basics** [comment]
I sent you a DM on gitter with a possible workaround. Is that enough?

**Comment by FArekkusu** [comment]
It worked after I additionally selected the `option` I modified in the dropdown.

I'll leave this issue open just in case; if you think that it's duplicate to the previously reported one, you can close it.

**Comment by Blind4Basics** [comment]
I talk about this in another issue (#1939), but I believe I never opened a dedicated issue. Couldn't get my hand on anything similar either, so let's keep this one.

**Comment by Blind4Basics** [comment]
I updated the related lis on the wiki: https://github.com/codewars/codewars.com/wiki/List-of-Kata-not-editable-anymore-due-to-rank-inconsistencies

@FArekkusu : could you plz update your first message to clearly specify that the problem comes from the "wild" reranking of the kata?
